rust/lance/src/index/vector.rs

```diff
@@ -130,6 +152,17 @@ fn is_ivf_pq(stages: &[StageParams]) -> bool {
         && matches!(&stages[len - 2], StageParams::Ivf(_))
 }
 
+fn is_ivf_hnsw_pq(stages: &[StageParams]) -> bool {
+    if stages.len() < 2 {
```

Comment:
Should this be `< 3`?

Reply:
have no idea how the test passed, just added assertions to make sure the index is created correctly